.gitignore

```diff
@@ -44,6 +44,7 @@ htmlcov/
 nosetests.xml
 coverage.xml
 _trial_temp*
+.hypothesis
```

Comment:
Thanks, I had meant to mention that in a previous review, but multi-tasking got the better of my brain.